src/App.tsx

```diff
@@ -15,11 +16,13 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
-}
+// enum SortType {
+//   NONE,
+//   ALPHABET,
+//   LENGTH,
+// }
+
+type SortType = 'none' | 'alphabet' | 'length' ;
```

Comment:
Interesting approach, you decided not to work with the enums, in case you wanted the enums you could reach them by using SortType.ALPHABET or SortType.LENGTH etc. but I dont believe it makes much difference, you can also create a type like you did.